packages/esm-patient-registration-app/src/patient-registration/patient-registration.component.tsx

```diff
@@ -158,9 +158,7 @@ export const PatientRegistration: React.FC<PatientRegistrationProps> = ({ savePa
           <div className={styles.formContainer}>
             <div>
               <div className={styles.stickyColumn}>
-                <h4>
-                  {inEditMode ? t('edit', 'Edit') : t('createNew', 'Create New')} {t('patient', 'Patient')}
-                </h4>
+                <h4>{inEditMode ? t('editPatient', 'Edit Patient') : t('createNewPatient', 'Create New Patient')}</h4>
```

Comment:
```suggestion
                <h4>{inEditMode ? t('editPatientDetails', 'Edit patient details') : t('createNewPatient', 'Create new patient')}</h4>
```

https://zpl.io/agm3dOA